src/hooks/useGetWishlist.js

```diff
@@ -0,0 +1,37 @@
+import { useEffect } from "react";
+import { useAuthContext } from "../context/AuthProvider";
+import axios from "axios";
+import { useStateContext } from "../context/StateProvider";
+
+const encodedToken = localStorage.getItem("token");
+
+export const useGetWishlist = (wishlist) => {
+  const { userState } = useAuthContext();
+  const { productsDispatch } = useStateContext();
+
+  useEffect(() => {
+    if (userState._id) {
+      (async () => {
+        try {
+          const response = await axios.get("/api/user/wishlist", {
+            headers: {
+              authorization: encodedToken,
+            },
+          });
+          console.log(response.status);
+          console.log(response.data);
```

Comment:
can remove console.logs() later (if not required anymore)

Reply:
yup, missed them. Thank you!